### PR TITLE
Episodes shows

### PR DESCRIPTION
**Pull request by fondberg**

### CODE REVIEW

**Comment by fondberg** [comment]
@KapJI  @icarome

File: custom_components/spotcast/__init__.py [hunk 1]
```diff
@@ -14,7 +14,7 @@
 from homeassistant.components.cast.helpers import ChromeCastZeroconf
 from homeassistant.components.spotify.media_player import KNOWN_SPOTIFY_DEVICES
 
-__VERSION__ = "3.5.0"
+__VERSION__ = "3.5.1"
```
Comment (KapJI) [inline custom_components/spotcast/__init__.py:17]: Do you really need this? Component verison should be in `manifest.json`.

File: custom_components/spotcast/__init__.py [hunk 2]
```diff
@@ -253,7 +253,17 @@ def play(client, spotify_device_id, uri, random_song, repeat, shuffle, position)
             uri,
             spotify_device_id,
         )
-        if uri.find("track") > 0:
+        if uri.find("show") > 0:
+            show_episodes_info = client.show_episodes(uri)
+            if show_episodes_info and len(show_episodes_info["items"]) > 0:
```
Comment (KapJI) [inline custom_components/spotcast/__init__.py:258]:
```suggestion
            if show_episodes_info and show_episodes_info["items"]:
```

**Review by KapJI — APPROVED** [review]

**Comment by fondberg** [comment]
accidently released with #178

File: custom_components/spotcast/__init__.py (hunk 2 above)
Comment (fondberg) [reply custom_components/spotcast/__init__.py:258]: Saw your review now! Thanks, I'm nackered now after a whole day fixing. Can you open a new PR?

**Pull request closed without merging**